### ISSUE TITLE
External issue tracking.

### ISSUE THREAD

**Issue by Maikuolan** [CIDRAM/CIDRAM#22]
*Will be marked off upon completion/resolution. Issue will be closed upon completion/resolution of all listed items.*

- [x] "[Hosters that should be added](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4039)" (addressing the two-tiered/multi-tiered approach for signature files and related updates; various suggestions for new ASN additions; some UA suggestions) - *Resolved (https://github.com/Maikuolan/CIDRAM/commit/7fd99d7ea377cc52adfbbb5f44dbcd283689b3ea).*
- [x] "[Yahoo bot false positive](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4041)" (persistent problems with search engine verification; likely lookup failures; need to investigate and resolve) - *Fix not necessary; See final comment of this issue.*
- [x] "[Implementing macmathan's browser blocks?](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4042)" (unexplained duplication of new module/component entries in `cidramblocklists.dat` file and some other strange behaviour; need to determine cause) - *Fixed (https://github.com/Maikuolan/CIDRAM/commit/c8778924df2cb94d40796a0876005c46b711e76e).*
- [x] "[Implementing badhost.inc?](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4037)" (completion of "bad hosts blocker module") - *Resolved (https://github.com/Maikuolan/CIDRAM-Extras/commit/eceec6118eae7e9af1927c3123ffd8e481421d71).*
- [x] "[[Installable Module] Optional security extras module](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4029)" (completion of "optional security extras module") - *Resolved (https://github.com/Maikuolan/CIDRAM-Extras/commit/daae67c1c6c94eefef40271c199b376697daa35d).*
- [x] "[Not blocking?](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4043)" (block event ID not being correctly logged) - *Fixed (https://github.com/Maikuolan/CIDRAM/commit/1b3745bc7bc4e8454d78e1471092b6ee8c426458).*
- [x] "[Traversal attacks from Bing](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4036)" (incorrect signature description) - *Fixed (https://github.com/Maikuolan/CIDRAM-Extras/commit/7653c5aff27de16201fa32fd11f2a07ec28ef5e6).*
- [x] "[Can't reinstall cidram](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4050)" (possible issue with `ban_override`; need to investigate further) - *Fixed (https://github.com/Maikuolan/CIDRAM/commit/ba6e1ec41e61992765943ef1f892e6eb6c818828).*
- [x] "[Whitelisting](http://www.spambotsecurity.com/forum/viewtopic.php?f=61&t=4076)" (bug: whitelisting an IP/CIDR via IPv4/IPv6 signature files does not currently result in whitelisting an IP from being blocked by modules) - *Fixed (https://github.com/Maikuolan/CIDRAM/commit/90f098ab773340c79fc207f855783b160e92ae91).*

**Comment by DanielRuf** [comment]
Would be good to have the actual commits linked which solve the points. ;-)

**Comment by Maikuolan** [comment]
Good point. :+1: 

I'll edit them in now.

**Comment by Maikuolan** [comment]
*New issue added: "Can't reinstall cidram".*

**Comment by Maikuolan** [comment]
*Resolved issue: "Can't reinstall cidram".*

**Comment by Maikuolan** [comment]
*Resolved issue: "Implementing macmathan's browser blocks?".*

**Comment by Maikuolan** [comment]
*Resolved issue: "Implementing badhost.inc?".*

**Comment by Maikuolan** [comment]
*New issue added: "Whitelisting".*

**Comment by Maikuolan** [comment]
*Resolved issue: "Whitelisting".*

**Comment by Maikuolan** [comment]
*Resolved issue: "Hosters that should be added".*

**Comment by Maikuolan** [comment]
*Resolved issue: "[Installable Module] Optional security extras module".*

**Comment by Maikuolan** [comment]
*Based on the results of debugging, the final issue, "Yahoo bot false positive", seems to be an issue isolated to the hosting environment for a specific user. Mainlining a fix for this shouldn't be necessary. Marking as resolved and closing this issue.*
